geopandas/io/tests/test_file.py

```diff
@@ -16,7 +16,7 @@
 
 import geopandas
 from geopandas import GeoDataFrame, read_file
-from geopandas.io.file import _detect_driver, _EXTENSION_TO_DRIVER
+from geopandas.io.file import _detect_driver, _EXTENSION_TO_DRIVER, FIONA_GE_19
```

Comment:
You can define FIONA_GE_19 again a bit below, we already have other version checks there as well (that is a bit duplicative, but currently the test is failing, I assume because at the moment you import this value it is still set to False?)

Reply:
Something like that does appear to be going on. I added it to the test import check section.